README.md

```diff
@@ -201,10 +234,40 @@ To include all the files in a directory, use the `*` wild card (eg. `/var/log/*`
 | `hostname`            | no       | `hostname`                     | `myhostname`                   | Hostname |
 | `input_file_path_n`   | no       | `unset`                        | `/var/log/*`                   | Input log file path. **If this variable is not specified, default log agent configuration file is created.** |
 
-Additionally, you can add the `--no_service` to omit the step of automatically creating `monasca-log-agent.service` in `/etc/systemd/system/`
+Additionally, you can add the `--no_service` to omit the step of automatically
+creating `monasca-log-agent.service` in `/etc/systemd/system/`
+
+### Uninstalling Logs agent
+Stop the service:
+```
+systemctl stop monasca-log-agent
+systemctl disable monasca-log-agent
+```
+
+In the following description, `[target_dir]` is the target directory specified
+at the time of running `log-agent-[version].run`.
 
-### Monasca-ui
+Delete obsolete files:
+```
+rm -f [target_dir]/
```

Comment:
Please change option from `-f` to `-rf`.

Reply:
Done